pkgs/tools/misc/grc/default.nix

```diff
@@ -1,7 +1,4 @@
-{ lib
-, fetchFromGitHub
-, buildPythonApplication
-}:
+{ lib, fetchFromGitHub, buildPythonApplication, installShellFiles }:
```

Comment:
Keep one argument per line.

The argument should be obvious after all those years: your diff has four lines, whereas adding `installShellFiles` in the next line generate only one